### ISSUE TITLE
:sparkles: Source Xero : Migrate Python CDK to Low-code CDK

### ISSUE THREAD

**Issue by RealChrisSean** [airbytehq/airbyte#28985]
https://docs.airbyte.com/integrations/sources/xero

**Rules for the Connectors Hackathon**
You can only work on one connector at a time
Make sure to resolve all suggestions from the Airbyte team
Must include all streams of the connector you’re migrating. Otherwise, it will not be accepted.
If you don’t commit anything within five days, the issue may be assigned to someone else.

**Comment by aazam-gh** [comment]
I'd like to take this one. I've issued a pr for the last one :D

**Comment by RealChrisSean** [comment]
Amazing!

**Comment by aazam-gh** [comment]
@RealChrisSean can i get the sandbox acc credientials?

**Comment by RealChrisSean** [comment]
Working on this! May take us a week or so, so please feel free to grab another issue while we wait on credentials.

**Comment by aazam-gh** [comment]
Alright will do :)

**Comment by aldre-strdm** [comment]
Happy to take this - as per above is there a Xero Sandbox/Developer account the Airbyte team can share to the contributor/me?

**Comment by justinbchau** [comment]
@marcosmarxm any thoughts on this?

**Comment by marcosmarxm** [comment]
It is possible to create a custom auth (basically reuse existing as a custom component) in lowcode.
Low code supports single use tokens so maybe give a try using the default component first.
For the credentials, please send a msg to me in Slack.

**Comment by aldrejota** [comment]
@marcosmarxm Sent you a Slack message. I could probably pull in this issue #31395 as well.
